### PR TITLE
BUG make sure to try a few times

### PR DESCRIPTION
**Pull request by beckermr**
cc @CJ-Wright

### CODE REVIEW

File: conda_forge_tick/cli.xsh [hunk 1]
```diff
@@ -26,13 +26,27 @@ def deploy(args):
...
+             'push',
+             'https://{token}@github.com/{deploy_repo}.git'.format(
+                 token=$PASSWORD, deploy_repo='regro/cf-graph-countyfair'),
+             'master'],
+             token =$PASSWORD.encode('utf-8'))
+
+        try += 1
+
+    if status != 0:
+        raise RuntimeError("bot did not push its data! stopping!")
```
Comment (CJ-Wright) [inline conda_forge_tick/cli.xsh:48]: push it to a branch?

Comment (beckermr) [reply conda_forge_tick/cli.xsh:48]: good idea

Comment (CJ-Wright) [reply conda_forge_tick/cli.xsh:48]: Also maybe halt itself in the circle_worker repo? (and ping us on all the commits so we can't ignore it?)

Comment (beckermr) [reply conda_forge_tick/cli.xsh:48]: I did that. Now the restart script only runs if deploy works

Comment (CJ-Wright) [reply conda_forge_tick/cli.xsh:48]: cool

Comment (beckermr) [reply conda_forge_tick/cli.xsh:60]: done

**Pull request merged**